tagstore/models/user.py

```diff
@@ -0,0 +1,7 @@
+"""
+User-related types
+"""
+
+from typing import NewType
+
+UserId = NewType('UserId', int)
```

Comment:
Bugs me that the `UserId` is an integer..  but I guess we can change this if/when we need to?

Reply:
What type would you suggest instead? String?

Reply:
Yep string -- since users in edx-platform are identified by the username, not the numeric ID.

Reply:
Check out a304535f296bfa73f1cf787fa45f9f3377bbcfa4 in which I used the `EntityId` models for users instead of an integer ID. I think that's more consistent with the design, and I'd always intended for the future possibility of tagging users by representing them with an `EntityId`.

Reply:
Oo that's lovely.. so even people are Entities!  I can see use cases for that, and lovely that even if they never come up in reality, it's no less or more work to maintain.

Great solution!